protocol-specs/generators/base_generator.js

```diff
@@ -33,7 +33,7 @@ const runGenerator = (generatorName, suiteCreators) => {
 
 		const fullPath = `${dirPath}/${suiteResult.handler}.json`;
 
-		fs.writeFileSync(fullPath, JSON.stringify(suiteResult, null, 2));
+		fs.writeFileSync(fullPath, `${JSON.stringify(suiteResult, null, 2)}\u000A`);
```

Comment:
why do we need to do this way? maybe comment would be nice

Reply:
Prettier was removing last line and linter was complaining about \n will look for workarounds or add comment.

Reply:
Fixed guess was my IDE being funky